js/pint-types-bundle/LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
symlink instead `ln -s ../../LICENSE` ?

Reply:
if we use `ln -s ../../LICENSE`, it will be

```
$ cat LICENSE
cat: LICENSE: Too many levels of symbolic links
```

Reply:
if we use `ln -s ../../LICENSE`, it will be

```
$ cat LICENSE
cat: LICENSE: Too many levels of symbolic links
```

Reply:
For example

https://app.circleci.com/pipelines/github/ChainSafe/PINT/290/workflows/e0620c11-c90d-4701-8e4e-e3a6e4a03b7c/jobs/1486